### PR TITLE
Add ReadHandler to handle the ReadCommand

### LINKED ISSUE #13: Command handler for the "Read" command
### Notice
If you want to do this issue, assign yourself to the issue so that we don't have three people submitting PRs for the same issue please :)

Blocked by #11.

### User story
As a user,
I want a read command
so that I can check the details of a task from my todo list.

### Acceptance criteria
- A new class that implements the `IHandler<ReadCommand>` interface.
- Uses the persistance interface from #11 to handle persistance.

If you can think of other things that adds value, feel free to add it.

### PR DESCRIPTION
**Pull request by pobiega**
closes #13

### CODE REVIEW

File: BusyList/Handlers/ReadHandler.cs [hunk 1]
```diff
@@ -3,11 +3,27 @@
...
         public void Run(ReadCommand command)
         {
-            Console.WriteLine($"If the ReadHandler was implemented, I would now list details for Task {command.Id}.");
+            var task = _taskRepository.GetTaskById(command.Id);
+
+            if (task == null)
+            {
+                Console.WriteLine($"No task found with id {command.Id}");
+            }
+            else
```
Comment (pobiega) [inline BusyList/Handlers/ReadHandler.cs:23]: Skip the else clause, use a `return;` in the null checked branch instead.

File: BusyList/Handlers/ReadHandler.cs [hunk 2]
```diff
@@ -3,11 +3,27 @@
...
+            {
+                Console.WriteLine($"The following task has been found: {task.Id}: {task.Description}");
```
Comment (pobiega) [inline BusyList/Handlers/ReadHandler.cs:25]: How about using the `task.Print()` method that was added for this purpose?

File: BusyList/Handlers/ReadHandler.cs [hunk 3]
```diff
@@ -16,14 +16,8 @@ public void Run(ReadCommand command)
...
 
-            if (task == null)
-            {
-                Console.WriteLine($"No task found with id {command.Id}");
-            }
-            else
-            {
-                Console.WriteLine($"The following task has been found: {task.Id}: {task.Description}");
-            }
+            if (task == null) return;
```
Comment (pobiega) [inline BusyList/Handlers/ReadHandler.cs:19]: Should let the user know that there was no task with the given ID and THEN return.

Comment (SquallyWally) [reply BusyList/Handlers/ReadHandler.cs:19]: But won't the second statement be unreachable?  Like this:
 ``` if (task == null) Console.WriteLine($"This task does not exist with the id {task.Id}"); return;
            Console.WriteLine(task.Print());```

Comment (pobiega) [reply BusyList/Handlers/ReadHandler.cs:19]:
```csharp
if (task == null)
{
	Console.WriteLine($"No task with id {command.Id} found.");
	return;
}

Console.WriteLine(task.Print());
```

Comment (SquallyWally) [reply BusyList/Handlers/ReadHandler.cs:19]: Weird last time when I did that it didnt worked properly. I'll try it again then.

**Review by denislavh — APPROVED** [review]

**Review by chrisK00 — APPROVED** [review]

**Pull request merged**